source/module_io/binstream.cpp

```diff
@@ -37,18 +37,12 @@ void Binstream::open(const std::string filename,const char *op)
 // we can use if(!Binstream) ...
 bool Binstream::operator!() const
 {
-	if (fileptr==NULL)
-		return true;
-	else
-		return false;
+	return fileptr==NULL;
```

Comment:
I  am a little bit confused here, why use `nullptr` and `NULL` in the same file? Seems more reasonable choice would be using `nulltpr` for all?

Reply:
> I am a little bit confused here, why use `nullptr` and `NULL` in the same file? Seems more reasonable choice would be using `nulltpr` for all?

Only one fix can be applied at once.

```suggestion
	return fileptr==nullptr;
```